packages/create-keystone-app/lib/get-example-project.js

```diff
@@ -1,13 +1,31 @@
 const prompts = require('prompts');
+const { getArgs } = require('./get-args');
 const { projects } = require('../example-projects/config');
 
 let EXAMPLE_PROJECT_CHOICE;
 
 const getExampleProject = async () => {
+  // If we already have the project template return it
   if (EXAMPLE_PROJECT_CHOICE) {
     return EXAMPLE_PROJECT_CHOICE;
   }
 
+  // If the project template was provided via the CLI arguments
+  const args = getArgs();
+  const argValue = args['--template'];
+  if (argValue) {
+    projects.map(project => {
+      if (project.folder == argValue) {
```

Comment:
It's a good idea to prefer strict equality here: `===` to ensure consistency with the rest of the codebase.

Reply:
Fixed in 42d96669